list.txt

```diff
@@ -16701,7 +16657,6 @@ koyocah.ml
 koyunum.com
 koyunum.net
 kozacki.pl
-kozow.com
```

Comment:
and this one (and so on, any domain related to the dyndns/no-ip and the like)

Reply:
add new commit - where SLD domain allowed